.github/workflows/e2e_with_cache.yaml

```diff
@@ -25,7 +25,6 @@ jobs:
                 php_version: ['8.1']
                 directory:
                     - 'e2e/applied-rule-removed-node-with-cache'
-                    - 'e2e/timeout-file-not-cached'
```

Comment:
This test is still needed.

Reply:
I see. Now it depends on copy-paste printer of hundred lines.

Could you add a new test with just few lines, not with the printer, that does the same effect?

Reply:
it can't only few lines, as it will succeed, which expect crash, it need a big file

Reply:
I see, rector can be configured with 0 second:

```php
$rectorConfig->parallel(0);
```

for it.

Reply:
I added it back with just few lines https://github.com/rectorphp/rector-src/pull/4051/commits/5a2cfeba698b36412b074fce63bb3b93f1ddbae7